packages/baml/baml_client/globals.ts

```
@@ -15,53 +15,62 @@ $ pnpm add @boundaryml/baml
-const patchedLoad = (originalFn: any) => (...args: any[]) => {
-    const result = originalFn(...args);
-    try {
-        // Dont fail if env vars fail to reset
-        resetBamlEnvVars(process.env);
-    } catch (e) {
-        console.error(e);
-    }
-    return result;
-};
+const patchedLoad =
+	(originalFn: any) =>
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)